components/scream/src/share/scream_pack.hpp

```diff
@@ -220,7 +256,7 @@ scream_pack_gen_unary_stdfn(tgamma)
 template <typename Pack> KOKKOS_INLINE_FUNCTION
 OnlyPackReturn<Pack, typename Pack::scalar> min (const Pack& p) {
   typename Pack::scalar v(p[0]);
-  vector_disabled for (int i = 0; i < Pack::n; ++i) v = util::min(v, p[i]);
+  vector_simd for (int i = 0; i < Pack::n; ++i) v = util::min(v, p[i]);
```

Comment:
Don't know why vectorization was disabled for min and not max. It wasn't like that in micro app.

Reply:
I'll check. The micro-app doesn't use the vectorized min, just the vectorized max. The unit test in the scream main repo shows that min is broken in I think Intel 18. I'll check for sure when I run this on a KNL or SKX testbed and switch it back if needed.